spec/presenters/petition_csv_presenter_spec.rb

```diff
@@ -34,6 +34,32 @@
         specify { is_expected.to eq(csvd_not_public_petition petition) }
       end
     end
+
+    describe "mappings" do
+      let(:petition) { FactoryBot.create(:completed_petition) }
+
+      describe "under_consideration_at" do
+        it "maps to the petition's closed_at" do
+          expect(PetitionCSVPresenter.fields).to include :under_consideration_at
+
+          index = PetitionCSVPresenter.fields.find_index :under_consideration_at
```

Comment:
this could be 

```ruby
index = PetitionCSVPresenter.fields.find_index :under_consideration_at

expect(index).to_not be_nil
```

But it's not as nice to whomever will break this code.

Reply:
Can't we do this using a spy? e.g

``` ruby
describe "mappings" do
  let(:presenter) { described_class.new(petition) }

  describe "under_consideration_at" do
    let(:petition) { FactoryBot.create(:closed_petition, closed_at: "20210302T12:30:00Z") }

    it "maps to the petition's closed_at" do
      allow(petition).to receive(:closed_at).and_call_original

      expect(presenter.under_consideration_at).to eq("20210302T12:30:00.000Z")
      expect(petition).to have_received(:closed_at)
    end
  end
end
```

Reply:
yeah although that doesn't assert that the right value is the right column

Reply:
That also applies to the setup you have - for a petition that doesn't collect signatures `open_at == closed_at` - so you could change `under_consideration_at` to `petition.send :open_at` and the test would pass.

Reply:
not sure I get it; you're asserting that: 

* the petition's method is called;
* the presenter returns that same value on the aliased method (although it could be forged since it's just a string).

whereas I'm checking that:

* the field is actually exported (i.e present in the CSV headers);
* the the row's field at that precise position matches the petition's field.

Happy to implement your version anyway, I just think mine is more precise?

Reply:
Probably misunderstood the intent of the description then - the `csvd_not_public_petition` method contains the fields in question so I assumed that the the intent of this spec was the mapping of closed_at to under_consideration_at. If the intent is as you described then happy to approve as is.